core-initrd/24.10/debian/control

```diff
@@ -0,0 +1,53 @@
+Source: ubuntu-core-initramfs
+Section: utils
+Priority: optional
+Maintainer: Dimitri John Ledkov <xnox@ubuntu.com>
```

Comment:
Should this change? This seems incorrect now

Reply:
Guess so. I've changed this in the control files now.